src/paillier.rs

```diff
@@ -80,6 +80,18 @@ impl PaillierEncryptionKey {
         let c = a.modmul(&b, self.0.nn());
         Ok((PaillierCiphertext(c), PaillierNonce(nonce)))
     }
+
+    /// Combines two [`PaillierNonce`]s into a [`MaskedNonce`] by computing
+    /// $`\text{nonce}_0 * \text{nonce}_1^{e} \text{mod} N`$, where $`N`$ is
```

Comment:
Huh, we've been using this in `swanky` but it looks like we use the `--html-in-header` with a script to enable latex support. And here I thought that was standard rustdoc behavior! Will fix.

Reply:
Oh, that's an interesting workaround. Rustdoc has avoided adding it as standard for a long time -- folks were working on it as far back as 2014. 
- https://github.com/rust-lang/rust/issues/34261
- https://github.com/rust-lang/rust/issues/16300
- https://github.com/rust-lang/rust/pull/17390#issuecomment-61195381

I might write up an issue about this for our repo. It would be very nice to be able to document all our mathematics more precisely. In the meantime, thanks for fixing.